linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitor.java

```diff
@@ -120,13 +120,9 @@ public static void main(String[] arguments)
         continue;
       }
 
-      if (path.isAbsolute()) {
-        // As of Guava 28, MoreFiles.fileTraverser returns relative paths. Just in case it changes the behavior,
-        // converting absolute paths to relative paths.
-        path = path.relativize(Paths.get(".").toAbsolutePath());
```

Comment:
The order was wrong. It must be `parentDirectory.relativize(child)`, where our parentDirectory is `projectDirectory`.